spec/system/candidate_interface/candidate_entering_degrees_spec.rb

```diff
@@ -257,6 +257,12 @@ def then_i_can_check_my_additional_degree
     expect(page).to have_content 'Masters Maths'
   end
 
+  def when_i_click_delete_on_my_additional_degree
+    within(find_all('.app-summary-card__header')[1]) do
```

Comment:
The additional degree appears first in list, so need [0] here i think.

Reply:
Hm for whatever reason locally it's showing up after the masters.

Reply:
![image](https://user-images.githubusercontent.com/42515961/85153861-9e39c200-b24e-11ea-998b-34b9e081ddac.png)

![image](https://user-images.githubusercontent.com/42515961/85153916-adb90b00-b24e-11ea-8de0-6dcc11bebad8.png)

Reply:
I'm assuming this is due to the undergrad degree being deleted and then added again causing an ordering issue.

Reply:
I'm seeing the same thing - Masters degree appearing consistently second in the list at the point the test fails.

Reply:
For some reason the test run on Azure seems to be failing because this order isn't deterministic. I would've thought implicit ordering by primary key would apply but apparently not. 

Getting a pass on Azure using the fix [here](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2322). Only remaining failure there is the Slack-related test that @duncanjbrown and @vigneshmsft are looking at.

Reply:
Great. Will close this then 👍